core/standalone/src/main/resources/standalone.conf

```diff
@@ -36,7 +36,7 @@ whisk {
   spi {
     ArtifactStoreProvider = "org.apache.openwhisk.core.database.memory.MemoryArtifactStoreProvider"
     MessagingProvider = "org.apache.openwhisk.connector.lean.LeanMessagingProvider"
-    LoadBalancerProvider = "org.apache.openwhisk.core.loadBalancer.LeanBalancer"
+    LoadBalancerProvider = "org.apache.openwhisk.core.loadBalancer.KindBasedLoadBalancer"
```

Comment:
A suggestion:
Rodric from OW suggested we call this more generic. I'm thinking at either:
* `MuxBalancer`
* `MultiplexerBalancer`


```javascript
loadbalancer {
  strategy {
     main:  org.apache.openwhisk.core.loadBalancer.MuxBalancer
     custom {
          kinds: {
            "nodejs": "org.apache.openwhisk.core.loadBalancer.TenantBasedBalancer"
            "others": "org.apache.openwhisk.core.loadBalancer.LeanBalancer"
         }
         annotations: {
            "max-performance": "org.apache.openwhisk.core.loadBalancer.TenantBasedBalancer"
            "others": "org.apache.openwhisk.core.loadBalancer.LeanBalancer"
         }
     }
  }
}
```